ansible/roles/setup-microshift-host/tasks/main.yml

```diff
@@ -7,6 +7,13 @@
     state: present
     update_cache: true
 
+- name: install golang
+  ansible.builtin.shell: |
```

Comment:
@sjug I know this is a little janky, but hopefully after upgrading to RHEL 8.7 we'll be able to revert this